src/addList.js

```diff
@@ -0,0 +1,7 @@
+export default class CreateList {
+  constructor(descrption, completed, index) {
+    this.description = descrption;
+    this.completed = completed;
```

Comment:
. It's better to have `completed = false` by default since no one adds a completed item to the list items.